interface/themes/oemr-bs3-override.css

```diff
@@ -0,0 +1,79 @@
+/*override bootstrap 3 for openemr and some new classes all prefixed with oe-*/
```

Comment:
Rec going through this file and categorizing each line into 2 categories:
1. If this is necessary and should never be modified by individual themes. If that is the case, then it should go into core.css.(note we should shoot for ultimate flexibility by each theme developer, so only stuff that is really required should go into this category)
2. Then the rest should be copy/pasted into the themes themselves and then the each theme developer has free reign on this stuff(where the buttons go, how btn-link looks, etc.). As an example, a theme developer may wish to go to bootstrap default legend/fieldset (ie. basically nothing) in their theme along with carnival music playing in the background(you just never know what the future styling fads will be :) ).